requirements.txt

```diff
@@ -7,4 +7,6 @@ pandas-stubs~=2.2.2
 pydantic~=2.7.1
 pytest~=7.2.1
 python-dateutil~=2.9.0
-requests~=2.31.0
+requests~=2.32.0
+antares~=0.3.25.0
```

Comment:
Don't know this package but I think you should not import it as i don't see where you use it